.github/label-actions.yml

```diff
@@ -3,7 +3,7 @@
     Hi there,
```

Comment:
We probably want to _keep_ the `auto:reproduction` label, even if we don't stale Discussions. That way we give a heads-up before _manually_ closing the Discussion as stale.